sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -602,9 +652,12 @@ class ContainerAPITest extends APISpec {
         blob.getProperties().getBlobSequenceNumber() == 0
         blob.getProperties().isServerEncrypted()
         blob.getProperties().isAccessTierInferred()
-        blob.getProperties().getAccessTier() == AccessTier.HOT
+        blob.getProperties().getAccessTier() == AccessTier.P10
```

Comment:
I'm guessing P10 is the default? It's determined by blob size right?

Reply:
https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types#premium-ssd